### DISCUSSION TITLE
使用 actions 打 windows 包，打不出来了，哪里不对吗？

### DISCUSSION THREAD

**Discussion by busyrat** [tw93/Pake#430] [Q&A]
使用 [Build App with Pake Cli](https://github.com/busyrat/Pake/actions/workflows/pake-cli.yaml) 构建的

提示 No files were found with the provided path: node_modules/pake-cli/output/*. No artifacts will be uploaded.

**Comment by Tlntin** [comment]
看起来解决了。

**Reply by busyrat** [reply]
是的，已经好了，有可能是icon地址有问题
